prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -14,7 +14,6 @@ import com.github.ajalt.clikt.parameters.types.int
 import com.github.difflib.text.DiffRow
 import com.github.difflib.text.DiffRowGenerator
 import com.github.doyaaaaaken.kotlincsv.dsl.csvReader
-import com.github.doyaaaaaken.kotlincsv.dsl.csvWriter
```

Comment:
Good move of 3rd party csv processor into Report Stream util class so that all the Commands class does not directly import from the 3rd party packages...